pyproject.toml

```diff
@@ -61,11 +61,11 @@ maintainers = [
 # Note: Python and NumPy upper version bounds should be set correctly in
 # release branches, see:
 #     https://scipy.github.io/devdocs/dev/core-dev/index.html#version-ranges-for-numpy-and-other-dependencies
-requires-python = ">=3.8"
+requires-python = ">=3.8,<3.11"
```

Comment:
I tried two different versions and got:

- ` packaging.specifiers.InvalidSpecifier: Invalid specifier: '>=3.8,<3.11'`
- `` packaging.specifiers.InvalidSpecifier: Invalid specifier: '>=3.8,<3.12'``

Unless I'm missing something obvious, an identical format worked just fine on the last release branch: https://github.com/scipy/scipy/blob/maintenance/1.8.x/pyproject.toml#L58

Is this through the meson stuff now and therefore different or?

Reply:
Let's make this `<3.12`, because in 1.9.x we're going to have to support Python 3.11 and people are already testing with it.

The first time we can/should upload wheels is the first bugfix release after 2022-08-01 (RC1 date), but we should allow building from source now.

Reply:
What to do about the traceback though?